src/components/Board.js

```diff
@@ -5,29 +5,95 @@ import axios from 'axios';
 import './Board.css';
 import Card from './Card';
 import NewCardForm from './NewCardForm';
-import CARD_DATA from '../data/card-data.json';
+// import CARD_DATA from '../data/card-data.json';
 
 class Board extends Component {
-  constructor() {
-    super();
+  constructor(props) {
+    super(props);
 
     this.state = {
-      cards: [],
+      url: this.props.url,
+      boardName: this.props.boardName,
+      cards: []
     };
   }
 
+  componentDidMount() {
+    const GET_INSPO_CARDS_URL = this.props.url + '/' + this.props.boardName + '/cards';
+
+    axios.get(GET_INSPO_CARDS_URL)
+    .then((response) => {
+      this.setState({
+        cards: response.data,
+      });
+    })
+    .catch((error) => {
+      this.setState({
+        error: error.message
+      });
+    });
+  }
+
+  deleteCard = (cardId) => {
+    const deleteURL = "https://inspiration-board.herokuapp.com/cards/" + cardId;
+
+    axios.delete(deleteURL)
+    .then((response) => {
+      const newCards = [...this.state.cards];
+      const index = newCards.findIndex(content => content.card.id === cardId);
+      newCards.splice(index, 1);
+
+      this.setState({
+        cards: newCards
+      });
+    })
+    .catch((error) => {
+      this.setState({
+        error: error.message
+      });
+    });
+  };
+
+  addCard = (cardData) => {
+    const POST_INSPO_CARDS_URL = this.props.url + '/' + this.props.boardName + '/cards';
+    axios.post(POST_INSPO_CARDS_URL, cardData)
+    .then((response) => {
+      cardData.id = response.data.card.id;
```

Comment:
I like that you've kept all the API interaction logic in one component - the callbacks are a little more complex, but I would say it makes the app as a whole much easier to comprehend. Whether or not you intended it, this is a great example of the container component pattern well-applied.